scipy/stats/_distn_infrastructure.py

```diff
@@ -100,7 +100,7 @@
     (Differential) entropy of the RV.
 """
 _doc_fit = """\
-fit(data, %(shapes)s, loc=0, scale=1)
+fit(data, %(shapes)s)
```

Comment:
```suggestion
fit(data)
```
The shape parameters are also optional. I think it's probably best to either
* remove all of these, as the only required argument is `data` or
* show all of the optional arguments including `floc`, `fscale`, and the appropriate `f0...` parameters, all with `None` as the default.